packages/terra-app-delegate/package.json

```diff
@@ -35,11 +32,11 @@
     "test": "npm run test:spec && npm run test:nightwatch-default",
     "test:spec": "$(cd ..; npm bin)/jest --config ../../jestconfig.json",
     "test:all": "npm run test:nightwatch-default && npm run test:nightwatch-chrome && npm run test:nightwatch-firefox && npm run test:nightwatch-safari",
-    "test:nightwatch-default": "WEBPACK_CONFIG_PATH=../../../../terra-site/webpack.config SPECTRE_TEST_SUITE=terra-app-delegate node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js",
-    "test:nightwatch-chrome": "WEBPACK_CONFIG_PATH=../../../../terra-site/webpack.config SPECTRE_TEST_SUITE=terra-app-delegate node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js chrome",
-    "test:nightwatch-firefox": "WEBPACK_CONFIG_PATH=../../../../terra-site/webpack.config SPECTRE_TEST_SUITE=terra-app-delegate node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js firefox",
-    "test:nightwatch-safari": "WEBPACK_CONFIG_PATH=../../../../terra-site/webpack.config SPECTRE_TEST_SUITE=terra-app-delegate node ../../node_modules/terra-toolkit/lib/scripts/nightwatch-non-parallel.js safari",
-    "test:remote": "WEBPACK_CONFIG_PATH=../../../../terra-site/webpack.config SPECTRE_TEST_SUITE=terra-app-delegate REMOTE=true node ../../node_modules/terra-toolkit/lib/scripts/nightwatch-process.js --config tests/nightwatch.conf.js",
-    "test:remote:all": "WEBPACK_CONFIG_PATH=../../../../terra-site/webpack.config SPECTRE_TEST_SUITE=terra-app-delegate REMOTE=true node ../../node_modules/terra-toolkit/lib/scripts/nightwatch-process.js --config tests/nightwatch.conf.js --env chrome-tiny,chrome-small,chrome-medium,chrome-large,chrome-huge,chrome-enormous,firefox-tiny,firefox-small,firefox-medium,firefox-large,firefox-huge,firefox-enormous,ie10-tiny,ie10-small,ie10-medium,ie10-large,ie10-huge,ie10-enormous,ie11-tiny,ie11-small,ie11-medium,ie11-large,ie11-huge,ie11-enormous,edge-tiny,edge-small,edge-medium,edge-large,edge-huge,edge-enormous,safari-tiny,safari-small,safari-medium,safari-large,safari-huge,safari-enormous"
+    "test:nightwatch-default": "SPECTRE_TEST_SUITE=terra-app-delegate node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js",
```

Comment:
Let's log an issue on the https://github.com/cerner/terra-toolkit for this work. We can tackle it in a separate PR. The scope of this PR is for moving terra-toolkit out of terra-core into it's own repo.